src/js/base/ffi-helpers.js

```diff
@@ -456,6 +456,8 @@ define(["js/runtime-util", "trove/lists", "trove/sets", "trove/option", "trove/e
 
         makeList: makeList,
         makeTreeSet: makeTreeSet,
+
+        isOption: runtime.getField(O, "is-Option"),
```

Comment:
Correct. You can click the unfold button under the line number, and you will see `isEither: runtime.getField(E, "is-Either"),`.